src/Tasks/BootstrapperUtil/BootstrapperBuilder.cs

```diff
@@ -447,7 +447,10 @@ public string[] GetOutputFolders(string[] productCodes, string culture, string f
             BuildPackages(settings, null, null, files, null);
 
             List<string> packagePaths = new List<string>() { invariantPath };
-            packagePaths.AddRange(Util.AdditionalPackagePaths.Select(p => Util.AddTrailingChar(p.ToLowerInvariant(), System.IO.Path.DirectorySeparatorChar)));
+            if (NativeMethodsShared.IsWindows)
```

Comment:
Is it right to have these small-scale checks rather than apply `SupportedOSPlatform` to the whole class?